### PR TITLE
Modifications for Ubuntu20.04 AMI

### PR DESCRIPTION
**Pull request by matesio**
Add packer configuration for Ubuntu 20.04
Disable Debian interactive mode error.
Modify setup-systemd function condition to match Ubuntu 20.04

### CODE REVIEW

**Review by kacperk — APPROVED** [review]
Looks ok

File: examples/consul-ami/consul.json [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
-      },
-      "owners": ["099720109477"],
-      "most_recent": true
-    },
-    "ssh_username": "ubuntu"
-  },{
-    "name": "ubuntu18-ami",
-    "ami_name": "consul-ubuntu-{{isotime | clean_resource_name}}-{{uuid}}",
-    "ami_description": "An Ubuntu 18.04 AMI that has Consul installed.",
+    "ami_description": "An Ubuntu 20.04 AMI that has Consul installed.",
```
Comment (martinkoch-geniebelt) [inline examples/consul-ami/consul.json:11]: Why was all the below deleted?

**Review by martinkoch-geniebelt — APPROVED** [review]
LGTM in general; just wondering why stuff was deleted

File: examples/consul-ami/consul.json (hunk 1 above)
Comment (matesio) [reply examples/consul-ami/consul.json:11]: We just need ubuntu 20 AMI, the original file generates three extra AMIs Amazon Linux, Ubuntu 18, and Ubuntu 16 so just removed the unnecessary configuration :).

**Pull request merged**